### PR TITLE
Feature/index

### PR DESCRIPTION
**Pull request by Alpha018**
WIP: comments in board component

### CODE REVIEW

File: src/app/index/index.component.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+          <div class="card" style="height: 280px">
+            <h5 class="card-header text-center box-color-header">
+              ESTADÍSTICAS
+            </h5>
+            <div class="card-body align-self-center" style="padding-top: 10px">
+              <div class="green">
+                <img src="https://hispachan.org/usuarios/usuariosonline.png" class="padding"> Online: {{stats?.online}}
+              </div>
+              <div class="normal">
+                <img src="https://hispachan.org/usuarios/usuarios.png" class="padding"> Annons Hoy: {{stats?.users}}
```
Comment (Gianfranco753) [inline src/app/index/index.component.html]:
```suggestion
                <img src="/assets/img/usuarios/usuarios.png" class="padding"> Annons Hoy: {{stats?.users}}
```

File: src/app/index/index.component.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+              </div>
+              <div class="normal">
+                <img src="https://hispachan.org/usuarios/posts.png" class="padding" style="margin-right: 10px">
```
Comment (Gianfranco753) [inline src/app/index/index.component.html]:
```suggestion
                <img src="/assets/img/usuarios/posts.png" class="padding" style="margin-right: 10px">
```

File: src/app/index/index.component.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                Posts: {{stats?.posts}}
+              </div>
+              <div class="text-center" style="margin: 10px 10px">
+                <img src="https://hispachan.org/hispaperro.gif">
```
Comment (Gianfranco753) [inline src/app/index/index.component.html]:
```suggestion
                <img src="/assets/img/hispaperro.gif">
```

File: src/app/index/index.component.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+                </ng-container>
+              </div>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+
+    <!--ONLY MOBIL-->
```
Comment (Gianfranco753) [inline src/app/index/index.component.html]:
```suggestion
    <!--ONLY MOBILE-->
```

File: src/app/index/index.component.html [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+              <span class="slug">{{'>>'}}</span>
+            </div>
+          </ng-container>
+        </div>
+      </div>
+
+      <div class="col-8 mx-auto">
+        <div class="box-stats-responsive">
+          <div class="green align-self-center">
+            <img src="https://hispachan.org/usuarios/usuariosonline.png" class="padding-responsive"> Online: {{stats?.online}}
```
Comment (Gianfranco753) [inline src/app/index/index.component.html]:
```suggestion
            <img src="/assets/img/usuarios/usuariosonline.png" class="padding-responsive"> Online: {{stats?.online}}
```

File: src/app/index/index.component.html [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+          </div>
+          <div class="normal-b align-self-center">
+            <img src="https://hispachan.org/usuarios/usuarios.png" class="padding-responsive"> Annons Hoy: {{stats?.users}}
```
Comment (Gianfranco753) [inline src/app/index/index.component.html]:
```suggestion
            <img src="/assets/img/usuarios/usuarios.png" class="padding-responsive"> Annons Hoy: {{stats?.users}}
```

File: src/app/index/index.component.html [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+          </div>
+          <div class="normal-c align-self-center">
+            <img src="https://hispachan.org/usuarios/posts.png" class="padding-responsive" style="margin-right: 10px">
```
Comment (Gianfranco753) [inline src/app/index/index.component.html]:
```suggestion
            <img src="/assets/img/usuarios/posts.png" class="padding-responsive" style="margin-right: 10px">
```

File: src/app/page404/page404.component.html [hunk 8]
```diff
@@ -0,0 +1,6 @@
+<div>
+  <h1>¡Página o hilo no encontrado! (Error 404)</h1>
+  <h2>¡Oops! La URL solicitada <b>{{url}}</b> no se encuentra en {{chan}} ;_;</h2>
+  <img class="rounded mx-auto d-block" src="https://www.hispachan.org/buttons/404.png"/>
```
Comment (Gianfranco753) [inline src/app/page404/page404.component.html]:
```suggestion
  <img class="rounded mx-auto d-block" src="/assets/img/buttons/404.png"/>
```

File: src/environments/environment.prod.ts [hunk 9]
```diff
@@ -1,3 +1,5 @@
 export const environment = {
-  production: true
+  production: true,
+  baseUrl: 'http://localhost:3000/api',
```
Comment (Gianfranco753) [inline src/environments/environment.prod.ts]: production runs on localhost?

**Comment by Alpha018** [comment]
The files (.spect.ts) is autogenerate for the command `ng generate component <name>` for this moment ignore that files.

**Pull request merged**